operations/patch.go

```diff
@@ -177,6 +174,14 @@ func Patch() cli.Command {
 			}
 			params.Description = params.getDescription()
 
+			if err = params.setLocalAliases(conf); err != nil {
```

Comment:
I can't put a comment on the actual line SOS but we should change the patch flag description now to say something about how aliases can be defined locally